### PR TITLE
oneDPL 2022.5.0 Release Notes Update

### PR DESCRIPTION
**Pull request by ValentinaKats**
oneDPL 2022.5.0 Release Notes Update
Signed-off-by: Valentina Kats valentina.kats@intel.com

### CODE REVIEW

**Comment by mmichel11** [comment]
Under existing issues, I believe we should add the following note which was recently identified and is a regression from the previous release:

Crashes or incorrect results may occur when using ``oneapi::dpl::reverse_iterator`` or ``std::reverse_iterator`` with the DPCPP backend.

File: documentation/release_notes.rst [hunk 1]
```diff
@@ -8,6 +8,69 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
 
+New in 2022.5.0
+===============
+
+New Features
+------------
+- Added new ``histogram`` algorithms for generating a histogram from an input sequence into
+  an output sequence representing either equally spaced or user-defined bins.
+  These algorithms are currently only available for device execution policies.
+- Added ability to avoid group reduction operations for 64-bit arithmetic data types.
```
Comment (akukanov) [inline documentation/release_notes.rst:19]: This is not really a feature but a workaround to the known issue described at lines 64-66. Maybe the best way to refer to it is this wording we coined earlier (supposed to be right after the problem description) :
```
  For a workaround, define the ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION macro to 1
  before including oneDPL header files.
```

Comment (akukanov) [reply documentation/release_notes.rst:19]: We might as well add something to the bug fix section, e.g. "Added a workaround for reduction algorithm failures with 64-bit data types; see the Known Issues below" or alike.

File: documentation/release_notes.rst [hunk 2]
```diff
@@ -8,6 +8,69 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
+- Fixed handling of ``permutation_iterator`` as input to oneDPL algorithms for a variety of
+  source iterator and permutation types which caused issues.
+- Fixed ``zip_iterator`` to be `sycl device copyable`_ for trivially copyable source iterator types.
+
+Known Issues and Limitations
+----------------------------
+New in This Release
+^^^^^^^^^^^^^^^^^^^
+- Crashes or incorrect results may occur when using ``oneapi::dpl::reverse_iterator`` or
+  ``std::reverse_iterator`` with device execution policies.
```
[the remark this answers is not in the document]
Comment (mmichel11) [reply documentation/release_notes.rst:33]: I agree with this suggested change.

File: documentation/release_notes.rst (hunk 1 above)
Comment (ValentinaKats) [reply documentation/release_notes.rst:19]: I added the workaround's description to the list of known issues. I also suggest to highlight it in the bug fix section, when it's addition is mentioned, to simplify the search.

**Review by akukanov — APPROVED** [review]

**Pull request merged**